test/plugin/test_buf_file.rb

```diff
@@ -1180,9 +1180,7 @@ def write_metadata(path, chunk_id, metadata, size, ctime, mtime)
   sub_test_case 'there are existing broken file chunks' do
     setup do
       @id_output = 'backup_test'
-      @bufdir = File.expand_path('../../tmp/broken_buffer_file', __FILE__)
-      FileUtils.rm_rf @bufdir rescue nil
-      FileUtils.mkdir_p @bufdir
+      @bufdir = Dir.mktmpdir
```

Comment:
hmm, `FileUtils.rm_rf` sometimes fails to execute if `@bufdir` already exists on Windows platform.
Therefore `Dir.mktmpdir` is used instead in here.

Reply:
This is because of the added test code.
The same instance is started multiple times within a single test.
Could you please fix the test `#resume backups enqueued broken metadata`?

Reply:
Then, we don't need to change this `setup`.